api/envoy/http/backend_routing/config.proto

```diff
@@ -22,23 +22,25 @@ message BackendRoutingRule {
   // Operation name, also known as selector.
   string operation = 1 [(validate.rules).string.min_bytes = 1];
 
-  // True if CONSTANT_ADDRESS, otherwise is APPEND_PATH_TO_ADDRESS.
+  // Corresponds to the path translation options in Google Cloud Endpoints.
   // https://cloud.google.com/endpoints/docs/openapi/openapi-extensions#understanding_path_translation
-  bool is_const_address = 2;
+  enum PathTranslation {
+    PATH_TRANSLATION_UNSPECIFIED = 0;
+    CONSTANT_ADDRESS = 1;
+    APPEND_PATH_TO_ADDRESS = 2;
+  }
+
+  PathTranslation path_translation = 2;
```

Comment:
Please don't re-use same index for backward compatibility

Reply:
Done